Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java

```diff
@@ -50,111 +55,133 @@
  */
 @Controller
 public class ConceptAddController {
-	
-	private static final Logger logger = LoggerFactory
-			.getLogger(ConceptAddController.class);
-
-	@Autowired
-	private IConceptManager conceptManager;
-	
-	@Autowired
-	private IConceptListManager conceptListManager;
-
-	@Autowired
-	private IConceptTypeManger conceptTypesManager;
-
-	@Autowired
-	private ServiceRegistry serviceRegistry;
-	
-	@Autowired
-	private IIndexService indexService;
-	
-	@Value("#{messages['INDEXER_RUNNING']}")
+
+    private static final Logger logger = LoggerFactory.getLogger(ConceptAddController.class);
+
+    @Autowired
+    private IConceptManager conceptManager;
+
```

Comment:
@jdamerow  
Tab policy is set to spaces only and spacing size is 4. 
Text file delimiter is set to Unix